src/borg/key.py

```diff
@@ -371,7 +371,7 @@ def encrypt_key_file(self, data, passphrase):
         iterations = PBKDF2_ITERATIONS
         key = passphrase.kdf(salt, iterations, 32)
         hash = hmac_sha256(key, data)
-        cdata = AES(is_encrypt=True, key=key).encrypt(data)
+        cdata = AES(key, b'\0'*16).encrypt(data)
```

Comment:
Yes, it is not in the scope of this PR. It‘s just that this PR makes the code obvious to see.

Why is this problematic? Maybe it is not in current borg, because we don‘t change the plaintext content. I have to think about what this means for change-passphrase (it reveals the xor of the two key streams at least). In future the keyblob will be editable to add new keys, i think we need to keep in mind that we need to come up with a solution at that time.